Sources/TootSDK/TootClient/TootClient.swift

```diff
@@ -340,4 +340,9 @@ extension TootClient {
         }
         self.flavour = instance.flavour
     }
+    
+    /// Returns `true` if this instance of `TootClient` has an `accessToken`.
+    public var isAuthorized: Bool {
```

Comment:
I think this field may be promising more than intended. Even if there is a cached access token, it doesn't mean it's a working token. The consuming app is better suited to perform this check with the correct intention as it will be more aware of the lifecycle of TootClient.

The best way to check if the client can be used for API calls is to invoke `verifyCredentials()` - if the method succeeds and returns the user's Account, then the current token is valid.

Reply:
Updated naming and implementation to better match my usage. I want to detect if a client is fully anonymous.